portfolio/src/main/webapp/script.js

```diff
@@ -43,20 +43,25 @@ function getRandomGreeting() {
  * @param {Array} - the list of comments
  */
 function printUserComments(comments){
-  for(let index = 0; index < comments.length; index++){
+  for(let index=0; index<comments.length; index++){
     var commentHolder = document.createElement("li");
     var comment = document.createTextNode(`${comments[index]}`);
     commentHolder.appendChild(comment);
     document.getElementById('Comments').appendChild(commentHolder);
   }
 }
 
+function deleteUserComments(){
+  fetch('/delete-comment', {method: 'POST'});
+  location.reload();
+  onloader();
+}
 /**
  * Fetches data from server and adds it to home page.
  */
-function onloader(){
+ function onloader(){
```

Comment:
I think you should tie this function to automatically trigger on window load, instead of having to explicitly call it.

Reply:
are you talking about in deleteUserComments() ?

Reply:
Oh I see what you mean